src/main/java/net/dv8tion/jda/api/entities/Guild.java

```diff
@@ -2169,6 +2294,141 @@ default Emote getEmoteById(long id)
         return retrieveMemberById(getOwnerIdLong());
     }
 
+    /**
+     * Load the member for the specified user.
+     * <br>If the member is already loaded it will be retrieved from {@link #getMemberById(long)}
+     * and immediately provided.
```

Comment:
No, it will update if the user information cannot be relied on due to intents. The cache is only used if it can be relied on. I'll make this more clear in the docs.